CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Safety Goggles
+
+Thanks for thinking of contributing!
+
+## We use Github
+
+Github is where we host code, track issues, and manage feature requests. Any additions or fixes to Safety Goggles are managed through pull requests.
+
+## Branching Strategies, deployment, and contributing.
+
+We use something similar to Github Flow, but we also maintain a dev branch. Anything pushed to main is automatically deployed to production. Currently, there's not a lot of testing, so the dev branch acts as a buffer for new releases. Pull requests are the best way to propose changes:
+
+1. Fork the repo and create your branch from `dev`.
+2. Make sure your code lints and builds (This is checked automatically in all PRs).
+3. run prettier `npm run prettier-format`
+4. [Issue your pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) against the `dev` branch!
+
+Eventually, there will be testing requirements, but not yet.
+
+## Any contributions you make will be under the MIT Software License
+
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project.
+
+## Bugs and Enhancements are handled with [Github issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues)
+
+If you'd like to report a bug or request a new feature/improvement [create an issue](https://github.com/devlyn37/Safety-Goggles/issues), labeling each as either a bug or an enhancement respectively.
+
```

Comment:
Maybe add a section on your roadmap and what you plan to add in the future?

Reply:
Yeah good idea